.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: csharp
 dist: trusty
-sudo: false
+sudo: true
```

Comment:
Sudo had to be enabled to be able to change PostgreSQL configuration to allow logical replication testing.